static/js/caseNotes.js

```diff
@@ -1,14 +1,15 @@
-$(function() {
+$(function () {
   var typeSelect = $('#type')
-  var subTypeSelect = $('#subType')
```

Comment:
this was a bug in the original code - the `select` has id `sub-type` instead so the sub type was never being disabled when the type select was chosen